src/sklearn_evaluation/plot/regression.py

```diff
@@ -9,8 +25,23 @@
 from sklearn.linear_model import LinearRegression
 
 
+def _set_ax_settings(ax, xlabel, ylabel, title):
+    ax.set_title(title)
+    ax.set_xlabel(xlabel)
+    ax.set_ylabel(ylabel)
+
+
+def _check_parameter_validity(y_true, y_pred):
+    if any((val is None for val in (y_true, y_pred))):
+        raise ValueError('y_true and y_pred are needed to plot '
+                         'Residuals Plot')
+
+    if y_true.shape != y_pred.shape:
+        raise ValueError('parameters should have same shape.')
+
+
 @SKLearnEvaluationLogger.log(feature='plot')
-def residuals(y_true, y_pred, ax=None, plot_name='Residuals Plot'):
+def residuals(y_true, y_pred, ax=None):
```

Comment:
any reason to remove the `plot_name` argument? I think it makes sense to remove it but I'm surprised that this PR contains such a change

Reply:
I saw this convention in most of the other plots. Also `ax` should be the last argument. Noticed this while adding the cook's distance so I thought of making this change in this PR itself.

Reply:
no need, we can take it as part of this!

Reply:
this is a special case cause the parameter is not doing much and I don't think many people are using this plot yet; however, we should always be careful or removing parameters as it changes the API and might break things for users

Reply:
Sure, will keep in mind